_rules/role-attribute-valid-value-674b10.md

```diff
@@ -5,11 +5,6 @@ rule_type: atomic
 description: |
   This rule checks that each `role` attribute has a valid value.
 accessibility_requirements:
-  wcag20:4.1.2: # Name, Role, Value (A)
-    forConformance: true
-    failed: not satisfied
-    passed: further testing needed
-    inapplicable: further testing needed
```

Comment:
I think we should fix the problem that this rule doesn't neatly map to WCAG, instead of not mapping it to WCAG at all. This rule becomes rather useless if we don't map it to any existing standard.

Reply:
Unfortunately, I don't see a SC we can easily map to.
Notably, it is not really possible to map to 4.1.2 since any element (not just UI component) can have a `role` and 4.1.2 is only concerned by UI components… and given the very nature of the rule, we cannot restrict it to "elements with a `role` inheriting from `widget`"…

So I've mapped to the ARIA (+modules) list of roles, which is the best I could find.
The `role` attribute can be used to many things. Its spec says that it's up to users to define the list of acceptable roles (https://www.w3.org/TR/role-attribute/#s_role_module_attributes), so ARIA list of roles is the best I could find.

This depends on https://github.com/act-rules/act-rules-web/pull/207 for working.